### PR TITLE
Faq video

### PR DESCRIPTION
**Pull request by Kyle199910**
Added Video to the FAQ page, test to see if it looks good as well as if the video works. Other minor changes include cover color of the questions.

### CODE REVIEW

**Comment by gaurav613** [comment]
A quick suggestion - instead of adding the full video, you can just embed the youtube link(https://www.youtube.com/watch?v=jlcd-_b6teo) in the div. Uploading the whole video will make the app very heavy, making it slow when deployed.

File: app/templates/FAQ.html [hunk 1]
```diff
@@ -14,7 +14,14 @@ <h1>Frequency Asked Questions(FAQ)</h1>
...
     </div>
 
+    <div class = "capstone">
+        <div class="intro_vid">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/jlcd-_b6teo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="border: 2px solid black;"></iframe>
+        </div>
+    </div>
+
     <div class="accordionMenu">
+        <p>Text1</p>
```
Comment (gaurav613) [inline app/templates/FAQ.html]: needs to be replaced with the proper text - there are two more tags like this below

File: app/templates/FAQ.html [hunk 2]
```diff
@@ -11,140 +11,135 @@
...
+                Tesla Powerwall product is used as the model for consumers to understand the benefits of having a BESS for the household because the Tesla Powerwall is the leading product in
+                the industry. It helps to simplify the results and present realistic numbers through an actual product on the market.
+              </div>
+          </div>
+
+          <p>Additional Links</p>
+          <a href="https://www.epa.gov/energy/greenhouse-gas-equivalencies-calculator" target="_blank">GHG Emission Equivalencies </a>
+          <a href="https://www.tesla.com/en_CA/support/powerwall/faqs" target="_blank">Tesla Powerwall Details </a>
+          <a href="https://www.oeb.ca/consumer-information-and-protection/electricity-rates" target="_blank">Ontario Electricity Rates </a>
+          <a href="https://www150.statcan.gc.ca/n1/daily-quotidien/220216/dq220216a-eng.html" target="_blank" style="margin-bottom:2%;">Inflation & Interest </a>
```
Comment (Rolo123y) [inline app/templates/FAQ.html]: Inflation & Interest link goes to a Error 404 page

**Review by Rolo123y — CHANGES_REQUESTED** [review]

File: app/templates/FAQ.html (hunk 2 above)
Comment (Kyle199910) [reply app/templates/FAQ.html]: @Rolo123y , the link has been updated

**Review by Rolo123y — APPROVED** [review]

**Comment by gaurav613** [comment]
Looks good to me, merging now.

**Pull request merged**